Dockerfile

```diff
@@ -166,7 +166,7 @@ ENV TESTING 1
 # Install development dependancies with poetry
 RUN set -ex \
  && apk add --no-cache --virtual .build-deps build-base \
- && poetry install --no-root \
+ && poetry install --no-root -E pg \
```

Comment:
How is the CI working without this right now? Doesn't the github action also build from this dockerFile?

Reply:
It does, but it uses the `libreg_active` target which had this correctly set, instead of the `libreg_local` target, which didn't have have this set (and has the `TESTING` env var set).